backend/src/test/java/com/huellapositiva/util/TestData.java

```diff
@@ -327,6 +327,10 @@ public JpaProposal registerESALAndNotPublishedProposal() {
         return registerESALAndProposal(UNPUBLISHED);
     }
 
+    public JpaProposal registerESALAndReviewPendingProposal() {
+        return registerESALAndProposal(REVIEW_PENDING);
+    }
+
```

Comment:
registerESALAndReviewXXXXXXProposal() methods are all redundant. Why not use registerESALAndProposal(XXXXXX) directly in the tests instead?

Reply:
I dont know, I saw that it was like this. It's refactored right now.